packages/cozy-harvest-lib/src/components/AccountsListModal.jsx

```diff
@@ -23,8 +23,8 @@ const AccountsListModal = ({ konnector, accounts, t }) => {
         <AccountsList
           accounts={accounts}
           konnector={konnector}
-          onPick={option => pushHistory(`/accounts/${option.account._id}`)}
-          addAccount={() => pushHistory('/new')}
+          onPick={option => replaceHistory(`/accounts/${option.account._id}`)}
+          addAccount={() => replaceHistory('/new')}
```

Comment:
I don't think we want a replace here... 

I think pushHistory is the right one

Reply:
Updated ✅